packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -278,25 +261,19 @@ export class IndexedDbPersistence implements Persistence {
   // is still experimental. When multi-client is switched to always on, `multiClientParams` will
   // no longer be optional.
   private constructor(
+    private readonly allowTabSynchronization: boolean,
     private readonly persistenceKey: string,
     private readonly clientId: ClientId,
     platform: Platform,
+    lruParams: LruParams,
     private readonly queue: AsyncQueue,
     serializer: JsonProtoSerializer,
-    lruParams: LruParams,
-    private readonly multiClientParams?: MultiClientParams
+    private readonly sequenceNumberSyncer?: SequenceNumberSyncer
```

Comment:
It looks like with this change, you are always passing a sequenceNumberSyncer, so this may not need to be optional?

Reply:
For IndexedDb it doesn't have to be optional anymore.